ocfweb/docs/docs/staff/tips/twitch.md

```diff
@@ -0,0 +1,35 @@
+[[!meta title="Using Twitch and OBS"]]
+
+## Setting Up Your Stream
+
+In your settings on Twitch, please enable VODs (video on demand) especially if you're working on OCF stuff.
+To do this, go to https://dashboard.twitch.tv/settings/channel and enable "Store past broadcasts".
+
+When streaming, keep the stream manager open (ideally on another screen) to see chat: https://dashboard.twitch.tv/stream-manager.
+
+## Setting up OBS to stream to Twitch
+
+OBS is pretty straightforward to configure. You can install it as you would any other software (arch pkg: obs-studio).
```

Comment:
link to their docs instead: https://obsproject.com/download

Reply:
https://obsproject.com/wiki/ too

Reply:
I said this on IRC, but it's worth noting that their docs want you to use a PPA on Ubuntu, which is not necessary. The page @dkess linked also only addresses Ubuntu. If we are going to link to them, we should use https://obsproject.com/wiki/Install-Instructions.